### PR TITLE
Issue 124 fixed: Form submits on enter after upload

### PR DESCRIPTION
**Pull request by SID177**
- After media is uploaded, the focus will be set to submit button, so enter event will click on submit button and form will be submitted
- Issue is solved on forum media upload, activity media upload, comment media upload and media page.

GL-124

### CODE REVIEW

File: app/assets/js/rtMedia.backbone.js [hunk 1]
```diff
@@ -536,6 +536,10 @@ jQuery( function( $ ) {
 				var upload_error = '';
 				var upload_error_sep = '';
 				var upload_remove_array = [ ];
+
+				var select_btn = jQuery( '.rtmedia-upload-input' );
```
Comment (milindmore22) [inline app/assets/js/rtMedia.backbone.js]: use `jQuery` or `$` consistently

File: package.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "name": "rtMedia",
   "dependencies": {
-    "load-grunt-tasks": "~0.2.1"
+    "load-grunt-tasks": "~0.2.1",
+    "npm": "^5.7.1"
```
Comment (milindmore22) [inline package.json]: `npm` locally ?

File: app/assets/js/rtMedia.backbone.js (hunk 1 above)
Comment (SID177) [reply app/assets/js/rtMedia.backbone.js]: `jQuery` is mostly used in this file, that's why I used it.
Do you want me to change this entire js to either `jQuery` or `$` or should I first confirm with Pranali?

File: package.json (hunk 2 above)
Comment (SID177) [reply package.json]: I didn't get this. I just ran `grunt` and `npm install`.

File: app/assets/js/rtMedia.backbone.js (hunk 1 above)
Comment (milindmore22) [reply app/assets/js/rtMedia.backbone.js]: It won't matter what you use, I recommended it for consistency.

File: package.json (hunk 2 above)
Comment (milindmore22) [reply package.json]: yes, please check why its got added

File: app/assets/js/rtMedia.backbone.js [hunk 3]
```diff
@@ -683,6 +687,29 @@ jQuery( function( $ ) {
...
 				}
 
+
+				/**
+				 *
+				 * Uploader improper enter behavior issue(124) fixed
+				 *
+				 */
+				var terms_conditions = $('#rtmedia_upload_terms_conditions');
+				if(terms_conditions.attr('id') === 'rtmedia_upload_terms_conditions'){
```
Comment (milindmore22) [inline app/assets/js/rtMedia.backbone.js]: whats use of this condition?

Comment (SID177) [reply app/assets/js/rtMedia.backbone.js]: Its same as `.length === 0`, for determining whether the element found or not.
Do you want me to change this to `.length === 0` condition?

Comment (milindmore22) [reply app/assets/js/rtMedia.backbone.js]: yes, please

File: package.json (hunk 2 above)
Comment (SID177) [reply package.json]: I think by mistake, I ran npm update command.
I've removed local `npm` as dependency and re-installed packages.

**Pull request closed without merging**